libsabergen/sabergen.py

```diff
@@ -50,42 +58,48 @@ def convert_to_ogg(self, song_path):
         
         try:
             # Setup new file name and path.
-            filename, prev_format_extension = path.splitext(path.basename(song_path))
+            filename, _ = path.splitext(path.basename(song_path))
             ogg_filename = "{}.ogg".format(filename)    
             ogg_path = path.join(path.dirname(song_path), ogg_filename)
 
-            # Convert with subprocess call
             # Command will generate a new file with the same name, except .ogg
-            # -c:a codec, -y overwrite if output exists, -q quality specifier
-            convert = subprocess.run(["ffmpeg","-y", "-i", song_path, "-c:a", "libvorbis", "-q:a", "4", ogg_path], stdout=subprocess.PIPE, stderr=subprocess.STDOUT )
+            subprocess.run("ffmpeg -y -i {} -c:a libvorbis -q:a 8 {}".format(song_path, ogg_path),
+                shell=True, capture_output=True, check=True)
```

Comment:
Pretty sure that capture_output breaks py3.5 as it's not supported yet.

Reply:
Good catch! Confirmed:
> Changed in version 3.7: Added the text parameter, as a more understandable alias of universal_newlines. Added the capture_output parameter.

Reply:
Fix committed and pushed. I also restored the comment about the individual flags - I had originally had the list form of args and spread that across lines to make flag->description relationships more clear but it ended up not working very well on my machine for some reason.